docs/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/index.md

```diff
@@ -0,0 +1,131 @@
+
+```js
+module.exports = {
+  plugins: [`gatsby-mdx`],
+}
+```
+
+Then we have another theme that handles applying a UI layer using to the core
+data model, `gatsby-theme-blog`. This theme is a child theme of
+`gatsby-theme-blog-core` and shadows the core theme's modules to render
+beautiful posts on each blog using any styling technology we want, like
+[Emotion](https://emotion.sh/) or [Sass](https://sass-lang.com/). Again, since
+this is a post on child theming itself we'll skip looking at how shadowing is
+applied to the parent and focus on one possible child theme `gatsby-config.js`.
+
+```js
+module.exports = {
+  __experimentalThemes: [`gatsby-theme-blog-core`],
+  plugins: [`gatsby-plugin-emotion`],
+}
+```
+
+Now that we've composed the original data model (with page creation logic) with
+the plugins and shadowed components to build the UI, any user can use our theme
+by specifying only the child in their `gatsby-config.js`.
+
+```js
+module.exports = {
+  __experimentalThemes: [`gatsby-theme-blog`],
+}
+```
+
+The end result from a theme's composition view is that we apply the parent, then
+the child, then the user's site. This opens up a whole host of additional
+possiblities as child themes can have multiple parents or compose with each
+other. Data models for blogging and ecommerce can be combined into a usable
+child theme and distributed as a single npm package. Users don't need to know
+your theme is made up of multiple parents.
+
+If we go back to our list of use cases, we can map each of them to a layer in
+this theme stack. The designer uses the child theme to make their own site
+(passing tokens into the theme config), the person proficient in HTML and CSS
+can shadow components from the core theme (using minimal React), and the Gatsby
+expert can craft custom data models for their specific use cases to compose with
+other models. Each link in the chain supports the next allowing us to develop a
+progressive disclosure of complexity for people with different skillsets.
+
+## Component Shadowing
+
+The other major change that has landed is the only breaking change to themes so
+far. Based on feedback from the early adopters who have been authoring themes,
+we've changed Component Shadowing so that it now applies to the entire src/
+directory instead of just the src/components directory. This change was made
+because of two key points of feedback.
+
+1. people had trouble remembering which folders got shadowed
+2. people who did remember, also wanted shadowing on templates and other, new
+   directories (for example one for design tokens) that didn't belong in
+   components
+
+The fix for any currently existing theme users is to move any shadowed
+components from `src/components/<theme-name>` to `src/<theme-name>/components`.
+
+## What's next?
+
+I'm working on a roadmap post for themes which will give more insight into the
+future of theming, how we're approaching stability, and what we're working on
+next.
```

Comment:
Can you wrap it up with a call to action or next step for the reader to take, even if that involves waiting for stuff to land?